### PR TITLE
Changelocatorandinfo

### PR DESCRIPTION
**Pull request by rpallavisharma**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

changed the locator and information document
### Description

modified js code
in both documents
### Motivation and Context

code got deleted. fixed a vaiable name
### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/hugo.toml [hunk 1]
```diff
@@ -5,7 +5,7 @@ enableRobotsTXT = true
 
 # Will give values to .Lastmod etc.
 enableGitInfo = true
-#ignoreErrors = ["error-remote-getjson"] 
+ignoreErrors = ["error-remote-getjson"]
```
Comment (harsha509) [inline website_and_docs/hugo.toml:8]: hi @pallavigitwork ,

This should be commented out.

File: website_and_docs/package-lock.json [hunk 2]
```diff
@@ -1319,42 +1452,49 @@
...
       "requires": {
         "queue-microtask": "^1.2.2"
       }
     },
     "slash": {
       "version": "5.0.0",
       "resolved": "https://registry.npmjs.org/slash/-/slash-5.0.0.tgz",
-      "integrity": "sha512-n6KkmvKS0623igEVj3FF0OZs1gYYJ0o0Hj939yc1fyxl2xt+xYpLnzJB6xBSqOfV9ZFLEWodBBN/heZJahuIJQ=="
+      "integrity": "sha512-n6KkmvKS0623igEVj3FF0OZs1gYYJ0o0Hj939yc1fyxl2xt+xYpLnzJB6xBSqOfV9ZFLEWodBBN/heZJahuIJQ==",
+      "dev": true
```
Comment (harsha509) [inline website_and_docs/package-lock.json:1485]: Strange,

How come all packages are added with `dev:true` and am not sure hugo packages are updated!

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @pallavigitwork ,

Thank you for the PR. PR looks good to me to get started, but could you please revert the files `hugo.toml, package.json and package-lock.json`.

**Comment by diemol** [comment]
@pallavigitwork is this PR still valid?

**Comment by rpallavisharma** [comment]
i will check and update you by this week.

**Pull request closed without merging**